.github/workflows/docs.yml

```diff
@@ -0,0 +1,107 @@
+name: Docs
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "docs/**"
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  docs-detect-changes:
```

Comment:
Maybe just `detect-changes`? We are already in the `Docs` workflow. This is also consistent with names we have in other repos.

Reply:
I agree it seems redundant to prefix the job name, but we do it to be able to distinguish them in GitHub. There are places where only the job name is used without the workflow context which may cause confusion on duplicates (e.g. in branch protection rules).
We do the same for `client-*` and `contracts-*` jobs in other workflows.